src/scancode/cli.py

```diff
@@ -131,6 +131,20 @@ def print_about(ctx, param, value):
 
     scancode -f json -l -c samples/zlib/ > scan.json
 
+Scan a directory while ignoring a single file. Print scan results to stdout as JSON:
+
+    scancode --ignore README samples/
+
+Scan a directory while ignoring all files with txt extension. Print scan results to
+stdout as JSON:
+
+    scancode --ignore *.txt samples/
```

Comment:
Are you sure this would work? a *nix shell would expand this pattern. Actually you should add a test doing just that and it will likely fail if there are more than one .txt file in the directory though the tests may be hard to exactly reproduce unquoted shell expansion. You can verify this in any case by trying this command your self with a dir that has more than one .txt file.